From 4327660c7154344a0c6075d5a665a4e5b89fc78d Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <perex@perex.cz>
Date: Tue, 28 Apr 2020 12:58:41 +0200
Subject: [PATCH 18/21] use-case.h: add USB as allowed device name

Some USB devices are really generic where the
PCM stream can be routed to different paths.

Signed-off-by: Jaroslav Kysela <perex@perex.cz>
---
 include/use-case.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/use-case.h b/include/use-case.h
index c2ba79a9fdb1..21df1de472d6 100644
--- a/include/use-case.h
+++ b/include/use-case.h
@@ -140,6 +140,7 @@ extern "C" {
 #define SND_USE_CASE_DEV_EARPIECE	"Earpiece"	/**< Earpiece Device */
 #define SND_USE_CASE_DEV_SPDIF		"SPDIF"		/**< SPDIF Device */
 #define SND_USE_CASE_DEV_HDMI		"HDMI"		/**< HDMI Device */
+#define SND_USE_CASE_DEV_USB		"USB"		/**< USB Device (multifunctional) */
 /* add new devices to end of list */
 
 
-- 
2.16.4

